src/main.rs

```diff
@@ -33,6 +33,11 @@ fn main() {
 	let mut last_height = 0;
 	for line in input.lines() {
 		let line = line.unwrap();
+		let line = if args.case_insensitive {
+			line.to_lowercase()
```

Comment:
Wow I was about to say that this is probably insufficient because it changes the output... but `uniq` seems to behave this way!
```
$ echo -e "HEY\nhey\nhEy" | sort | uniq -c -i
      3 hey
```